Makefile

```diff
@@ -0,0 +1,73 @@
+USER_ID := $(shell id -u)
+GROUP_ID := $(shell id -g)
+
+GIT_USER_ID := fybrik
+GIT_REPO_ID := datacatalog-go
+GIT_REPO_ID_MODELS := datacatalog-go-models
+GIT_REPO_ID_CLIENT := datacatalog-go-client
+
+DOCKER_HOSTNAME ?= ghcr.io
+DOCKER_NAMESPACE ?= fybrik
+DOCKER_TAG ?= 0.0.0
+DOCKER_NAME ?= openmetadata-connector
+
+FYBRIK_VERSION ?= v1.0.1
+
+TMP_FILE = tmpfile.tmp
+
+IMG := ${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/${DOCKER_NAME}:${DOCKER_TAG}
+export HELM_EXPERIMENTAL_OCI=1
+
+
+all: generate-code patch
+
+.PHONY: compile
```

Comment:
Please comment out/remove the unused rules. we can add them gradually with relevant code

Reply:
done